docs/user/collecting-metrics/prometheus-input.md

```diff
@@ -0,0 +1,94 @@
+| `prometheus.io/port` (mandatory)   | 8080, 9100 (no default value)                                    | Specify the port on the Pod where your application exposes metrics.                                                                |
+| `prometheus.io/path`               | /metrics (default), /custom_metrics                              | Set the HTTP path for the metrics endpoint.                                                                                        |
+| `prometheus.io/scheme`             | https-metrics (default with Istio), http (default without Istio) | Define the protocol for scraping: Either HTTPS with mTLS, or plain HTTP.                                                           |
+| `prometheus.io/param_<name>`       | Example: format: prometheus (no default)                         | Add a URL parameter to the scrape request. For example, prometheus.io/param_format: prometheus adds ?format=prometheus to the URL. |
+
+For example, see the following `Service` configuration:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "8080"
+    prometheus.io/scrape: "true"
+  name: sample
+spec:
+  ports:
+  - name: http-metrics
+    appProtocol: http
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: sample
+  type: ClusterIP
+```
+
+## Scrape Metrics from Istio-enabled Workloads
+
+If your application is part of an Istio service mesh, you must consider service port naming and mutual TLS (mTLS) configuration:
+
+- Istio must be able to identify the **appProtocol** from the Service port definition. Otherwise, Istio may block the scrape request.
+  You must either prefix the port name with the protocol like in `http-metrics`, or explicitly define the **appProtocol** attribute.
+
+- The metric agent can scrape endpoints from workloads that enforce mutual TLS (mTLS). For scraping through HTTPS, Istio must configure the workload using STRICT mTLS mode.
+  If you can't use STRICT mTLS mode, you can set up scraping through plain HTTP by adding the following annotation to your Service: `prometheus.io/scheme: http`. For related troubleshooting, see [MetricPipeline: Failed to Scrape Prometheus Endpoint](../troubleshooting.md#metricpipeline-failed-to-scrape-prometheus-endpoint).
+
+## Collect Diagnostic Metrics
+<!-- identical section for Prometheus and Istio docs -->
+To validate or debug your scraping configuration for the **prometheus** and **istio** input, you can use diagnostic metrics. By default, they are disabled.
+
+> **Note:** Unlike the **prometheus** and **istio** inputs, the **runtime**  input gathers data directly from Kubernetes APIs instead of using a scraping process, so it does not generate scrape-specific diagnostic metrics.
```

Comment:
```suggestion
> [!NOTE] 
> Unlike the **prometheus** and **istio** inputs, the **runtime**  input gathers data directly from Kubernetes APIs instead of using a scraping process, so it does not generate scrape-specific diagnostic metrics.
```